docs/users/installation.md

```diff
@@ -3,69 +3,95 @@ id: installation
-| @VERSION@        | @SCALAMETA@                   | @SCALA211@ / @SCALA212@ |
+| Scalafix  | Scala Compiler          | Scalameta   |
+| --------- | ----------------------- | ----------- |
+| 0.5.10    | 2.11.12 / 2.12.4        | 2.1.7       |
+| @VERSION@ | @SCALA211@ / @SCALA212@ | @SCALAMETA@ |
+
+Next, integrate Scalafix with your build tool or use the command-line interface.
 
 ## sbt
 
-The quickest way to get started is to run a **syntactic** rule like
-`ProcedureSyntax`
+Start by installing the sbt plugin in `project/plugins.sbt`
 
 ```scala
 // project/plugins.sbt
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "@VERSION@")
 ```
 
-From the sbt shell, invoke the `scalafix` task
+From the sbt shell, let's run a **syntactic** rule `ProcedureSyntax`
+
+```
+> myproject/scalafix ProcedureSyntax
+```
+
+If your project uses the deprecated "procedure syntax", you should have a diff
+in your sources like this
 
+```diff
+-  def unloadDrivers {
++  def unloadDrivers: Unit = {
 ```
-> myproject/scalafix ProcedureSyntax      // run rule on main sources
-> myproject/test:scalafix ProcedureSyntax // run rule on test sources
+
+Next, if we run a **semantic** rule like `RemoveUnusedImports` then we get an
+error
+
+```
+> myproject/scalafix RemoveUnusedImports
+[error] SemanticDB not found: META-INF/semanticdb/src/main/scala/...
 ```
 
-To run **semantic** rules like `RemoveUnusedImports` you need additional build
-configuration.
+The error message "SemanticDB not found" means the
+[SemanticDB](https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md)
+compiler plugin is not enabled in the build. Let's fix that by adding the
+following settings to `build.sbt`
 
 ```scala
 // build.sbt
 lazy val myproject = project.settings(
   scalaVersion := "@SCALA212@", // or @SCALA211@
   addCompilerPlugin(scalafixSemanticdb),
   scalacOptions ++= List(
-    "-Yrangepos",          // required by semanticdb-scalac compiler plugin
+    "-Yrangepos",          // required by SemanticDB compiler plugin
     "-Ywarn-unused-import" // required by `RemoveUnusedImports` rule
   )
 )
 ```
 
-Run semantic rules from the sbt shell like normal
+We run `RemoveUnusedImports` again and the error should be gone
 
 ```
 > myproject/scalafix RemoveUnusedImports
 ```
 
-Beware that the `semanticdb-scalac` compiler plugin adds around 7-25% overhead
-to compilation. It's recommended to provide generous JVM memory and stack
-settings in `.jvmopts`:
+If your project has unused imports, you should see a diff like this
 
+```diff
+- import scala.util.{ Success, Failure }
++ import scala.util.Success
 ```
--Xss8m
--Xms1G
--Xmx4G
-```
+
+Great! You are all set to use Scalafix with sbt :)
+
+> Beware that the `semanticdb-scalac` compiler plugin adds around 7-25% overhead
+> to compilation. It's recommended to provide generous JVM memory and stack
+> settings in `.jvmopts`:
+>
+> ```scala
```

Comment:
not scala

Reply:
Fair enough, I added it since it still rendered more nicely with the comment. Removed